test/utils-tests.jl

```diff
@@ -544,6 +544,20 @@ end
     [-0.37388888888888894, -1.0, -3.0]
 @test ten.([12.0, -0.0], [24, 30]) == ten.([" 12::24", " -0:30: "]) == [12.4, -0.5]
 
+#Test tic_once
+let
```

Comment:
Roger that.
So to summarize,there are three main new features that are to be implemented in all the new functions from here on (and that is to be added in all the existing functions). These are -
- Using @testset to start a set of tests
- Referencing other functions when required via [](@ref)
- Using prompt pasting to copy/paste code in the example section of the files